VagrantBuild.sh

```diff
@@ -38,7 +38,7 @@ if [ ! -f LocalConfig.pri ] && ! grep --quiet QMAKE_CXX LocalConfig.pri; then
 fi
 
 echo "Building Hoot..."
-make -s clean && make -sj$(nproc)
+make -s clean && make -j$(nproc)
 echo "Generating Eclipse project files..."
 make -s eclipse
 
```

Comment:
You meant to make this change?

Reply:
Yeah I did because the hanging Jenkins problem I turned on the verbose output to see if that would help us resolve this issue.